core/src/scene/styleContext.cpp

```diff
@@ -310,8 +315,10 @@ bool StyleContext::parseStyleResult(StyleParamKey _key, StyleParam::Value& _val)
             default:
                 break;
         }
+    } else if (duk_is_null_or_undefined(m_ctx, -1)) {
+        // Ignore setting value
```

Comment:
Can we put a `LOGD` here so logging can be switched back on when we're feeling paranoid?